### DISCUSSION TITLE
旧Siv3DのMIDI機能でCH10を指定してもパーカッションがならない件

### DISCUSSION THREAD

**Discussion by kestrel-90r** [Siv3D/OpenSiv3D#548] [Q&A] [answered]
OpenSiv3Dではないんですけど、MIDI機能でCH10を指定してもパーカッションがならないのですが対処方法あるでしょうか？
if (Input::KeyC.clicked) Midi::SendMessage(MidiMessage::SetInstrument(10, instrument.second));

**Comment by Reputeless** [comment, marked as answer]
🎉 OpenSiv3D GitHub Discussion 初質問 🎉

Siv3D の API が少し紛らわしくて申し訳ないのですが、
`MidiMessage::SetInstrument()` で指定するチャンネルは CH1 = 0, CH2 = 1, のゼロオリジンです。
CH10 の場合は `MidiMessage::SetInstrument()` の第一引数を `9` にします。

**Comment by kestrel-90r** [comment]
おお！そういうことですが。ありがとうございます。言われてみれば確かにそうですね！
